### DISCUSSION TITLE
IPF map looks rotated

### DISCUSSION THREAD

**Discussion by karthikp161985** [mtex-toolbox/mtex#1225] [Ask Anything]
Dear Ralf,
I have an EBSD map from a rolled sample from TSML OIM. I did the scanning such that the rolling direction is aligned along the A1 axis. 

I am using the following setting to take care of the coordinates in MTEX

ebsd = EBSD.load(fname,CS,'interface','ang',... 'convertEuler2SpatialReferenceFrame','setting 2');

The plotting convention are as follows in MTEX 

setMTEXpref('xAxisDirection','north');
setMTEXpref('zAxisDirection','outOfPlane');

Actually, I expect the rolling direction to be aligned along the north (vertical direction) in MTEX. But unfortunately, I see the rolling direction aligned along the horizontal direction. It looks like the IPF map is rotated 90 degrees about the screen normal. Could you please explain why such an issue happening? I request you to help me if possible. How can I plot the IPF map so that the rolling direction aligned along the north?  I have gone through this link but I couldn't understand 
https://mtex-toolbox.github.io/EBSDReferenceFrame.html
Also, Please let me know if you need more details from me.

**Comment by zmichels** [comment]
Was the X-axis possibly aligned horizontal when data were acquired (with rolling direction to north)? On one case you are talking about the coordinate reference frame, and the other you are talking about texture/deformation reference frame. Maybe try setting X to East and see if that works.

**Reply by karthikp161985** [reply]
Dear Michels, You are right. After changing to plotxeast I see the rolling direction (grains) along the vertical direction in the IPF map. But pole figures are rotated by 90 degrees about the z.axis. What should I do? Could you please suggest? The texture has changed now.

**Comment by zmichels** [comment]
Hmmm… if you remove the convertEuler2SpatialReferenceFrame… does that make it work? Probably not, but worth a shot. Ultimately, you *may* need to rotate the data at import… and try different configurations (I.e., keeping vs notKeep XY).

Zach
